### PR TITLE
Add support for the Solon framework

### PR DESCRIPTION
**Pull request by xsShuang**

### CODE REVIEW

File: oap-server/server-starter/src/main/resources/component-libraries.yml [hunk 1]
```diff
@@ -523,6 +523,9 @@ Sybase-jdbc-driver:
 OceanBase-jdbc-driver:
   id: 157
   languages: Java
+SolonMvc:
```
Comment (wu-sheng) [inline oap-server/server-starter/src/main/resources/component-libraries.yml:526]:
```suggestion
SolonMVC:
```

**Comment by wu-sheng** [comment]
You should not delete the PR template, you should follow it to update changes.

**Comment by xsShuang** [comment]
-[ x ]  Add a plugin that supports the Solon framework https://github.com/apache/skywalking-java/pull/697

**Comment by wu-sheng** [comment]
Change logs are still not updated.

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 * Increase `SW_CORE_GRPC_MAX_MESSAGE_SIZE` to 50MB.
 * Support to query relation metrics through PromQL.
 * Support trace MQE query for debugging.
+* Add support for the Solon framework.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:15]:
```suggestion
* Add Component ID(158) for the Solon framework.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**